gen3_tracker/meta/entities.py

```diff
@@ -286,13 +289,13 @@ def identifiers(self) -> dict:
         elif identifiers_len == 1:
             return {"identifier": identifiers[0].get("value")}
         else:
-            base_identifier = {"identifier": identifiers[0].get("value")}
-            base_identifier.update(
-                {
-                    identifier.get("system").split("/")[-1]: identifier.get("value")
-                    for identifier in identifiers[1:]
-                }
-            )
+            # Todo: Raise an execption if there are multiple identifiers with a "-" in them
+            base_identifier = {
+                "identifier" if "-" in identifier.get("system", "").split("/")[-1]
+                else identifier.get("system").split("/")[-1]: identifier.get("value")
```

Comment:
how do you know that - is the identifier that we should use?

Reply:
Like most things in the data-framer, it's pattern following from looking at the data.